core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20210916_clone_workspace_file_transfer.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="mtalbott" id="create_CLONE_WORKSPACE_FILE_TRANSFER">
+        <createTable tableName="CLONE_WORKSPACE_FILE_TRANSFER">
+            <column name="DEST_WORKSPACE_ID" type="BINARY(16)">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="SOURCE_WORKSPACE_ID" type="BINARY(16)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="COPY_FILES_WITH_PREFIX" type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet logicalFilePath="dummy" author="mtalbott" id="create_CLONE_WORKSPACE_FILE_TRANSFER_FK">
+        <addForeignKeyConstraint baseColumnNames="DEST_WORKSPACE_ID" baseTableName="CLONE_WORKSPACE_FILE_TRANSFER" constraintName="FK_CWBT_DEST_WORKSPACE" deferrable="false" initiallyDeferred="false" onDelete="CASCADE" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="WORKSPACE"/>
+        <addForeignKeyConstraint baseColumnNames="SOURCE_WORKSPACE_ID" baseTableName="CLONE_WORKSPACE_FILE_TRANSFER" constraintName="FK_CWBT_SOURCE_WORKSPACE" deferrable="false" initiallyDeferred="false" onDelete="CASCADE" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="WORKSPACE"/>
+    </changeSet>
+
+    <changeSet logicalFilePath="dummy" author="mtalbott" id="add_completed_clone_file_transfer_to_workspace">
+        <addColumn tableName="WORKSPACE">
+            <column name="COMPLETED_CLONE_WORKSPACE_FILE_TRANSFER" type="BIT(1)" defaultValue="1"/>
```

Comment:
If this represents that the work is done, perhaps use a time stamp instead? This would tell us when it was done as well as the fact it was done.